src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,117 @@
+.contact {
+  @include grid;
+  row-gap: 48px;
+
+  &__title {
+    @include sectionTitle;
+    margin: 0;
+    grid-column: 1/ -1;
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-column: span 5;
+    }
+  }
+
+  &__input {
+    // box-sizing: border-box;
+    background-color: $input-bg;
+    border-radius: 40px;
+    padding: 14px 16px;
+
+    font-size: 14px;
+    font-weight: 400;
+    color: $input-color;
+    font-family: "Poppins", sans-serif;
```

Comment:
font-family should be added to `html` and parent elements, if it needs to be changed for some blocks
that you can use `font-family: inherit`, as some properties inherited by default, some of them no